sdf/final-project/presentation.md

```diff
@@ -1,3 +1,8 @@
+<!-- 
+TODO:
+  - provide some guidelines on how to proctor final presentations (eg when to give 1/2 points)
```

Comment:
- Suggestion: We can all vote "yes" or "no" for a point, and if the votes are split in half we can give 1/2 points?